docs/pages/repo/docs/core-concepts/caching/to-cache-or-not-to-cache.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: To Cache or Not to Cache
+description: Whether 'tis more performant to cache.
+---
+
+import Callout from "../../../../../components/Callout";
+
+# To Cache or Not To Cache
+
+In building a monorepo you need to decide on a per-task basis if you are able to skip the task based upon having run it previously. This behavior is controlled via `pipeline.<task>.cache` inside of `turbo.json`:
```

Comment:
Do we want this to be focused on monorepo? It's outside the monorepo section in core concepts, and caching is not a monorepo-specific thing